styles.css

```diff
@@ -0,0 +1,172 @@
+body {
+  margin: 0;
+  padding-left: 24px;
+  padding-right: 24px;
+  background-color: #eceef1;
+  font-family: AvenirNext, Avenir, "Helvetica Neue", Helvetica, Arial,
+    sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+ul {
+  list-style-type: none;
+}
+fieldset {
+  border: 0px;
+  display: flex;
+  flex-direction: column;
+  width: 85%;
+  margin-bottom: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+label {
+  padding-bottom: 8px;
+}
+
+input {
+  height: 30px;
+  width: 100%;
+}
+
+input:focus {
+  border: 1px solid #58a2ec;
+  outline: 0;
+  border-color: rgba(82, 168, 236, 0.8);
+  box-shadow: 0 1px rgba(0, 0, 0, 0.075) inset, 0 0 8px rgba(82, 168, 236, 0.6);
+}
+
+.body-text {
+  font-size: 0.75rem;
+  color: #505050;
+}
+
+.body-text-link {
+  color: #0077c5;
+  text-decoration: none;
+}
+
+.body-text-link:hover {
+  text-decoration: underline;
+}
+
+.center {
+  justify-content: center;
+}
+
+.center-vertical {
+  align-items: center;
+}
+
+.col {
+  flex-direction: column;
+  align-items: center;
+}
+
+.container {
+  display: flex;
+  flex: 1;
+}
+
+.feature {
+  margin-left: 4px;
+  margin-right: 4px;
+  align-items: center;
+}
+
+.form-container {
+  background: white;
+  width: 44%;
+  border: 1px solid #c7c7c7;
+  padding-top: 42px;
+}
+.footer-text{
+ text-align: center;
+ margin-top: 24px;
+ margin-bottom:24px;
+ line-height: 1.5;
+ font-weight: 400;
+
+
+}
+
+.icon {
+  color: white;
+  margin-right: 8px;
+  background-color: cadetblue;
+  padding: 6px;
+  border-radius: 50px;
+}
+
+.lock-icon {
+  margin-right: 8px;
+}
+
+.subheadertext {
+  font-family: AvenirNext, Avenir, "Helvetica Neue", Helvetica, Arial,
+    sans-serif;
+  font-size: 1.25rem;
+  margin: 0;
+}
+
+.title {
+  margin: 0;
+  font-size: 24px;
+  color: #4c4c4c;
+  text-align: center;
+  font-weight: 400;
+}
+
+#logo {
+  color: grey;
+  margin: 0;
+}
+
+#navigation {
+  align-items: center;
+  justify-content: flex-end;
+}
+
+#signin-button {
+  border-radius: 5px;
+  border: 1px solid black;
+  height: 36px;
+  min-width: 90px;
+  background-color: #0077c5;
+  color: white;
+}
+
+#signup-button {
+  border-radius: 5px;
+  border: 1px solid black;
+  height: 44px;
+  width: 85%;
+  background-color: #0077c5;
+  color: white;
+  margin-bottom: 48px;
+}
+
+#tos-ps-text {
+  font-size: 0.75rem;
+  color: #505050;
+  font-style: italic;
+  text-align: center;
+  line-height: 1.5;
+  margin-bottom: 36px;
+}
+
+#captcha-text {
+  width: 100%;
+  text-align: center;
+  padding-top: 16px;
+  padding-bottom: 16px;
```

Comment:
Line 165 and 166 can best be replaced as padding: 16px 0;
Don't Repeat Yourself (D.R.Y)